src/OfficeService.Validation/WorkspaceType/EditWorkspaceTypeRequestValidator.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentValidation;
+using FluentValidation.Validators;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.OfficeService.Data.WorkspaceType.Interfaces;
+using LT.DigitalOffice.OfficeService.Models.Dto.Enums.Workspace;
+using LT.DigitalOffice.OfficeService.Models.Dto.Requests.WorkspaceType;
+using LT.DigitalOffice.OfficeService.Validation.WorkspaceType.Interfaces;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+
+namespace LT.DigitalOffice.OfficeService.Validation.WorkspaceType;
+
+public class EditWorkspaceTypeRequestValidator : BaseEditRequestValidator<EditWorkspaceTypeRequest>, IEditWorkspaceTypeRequestValidator
+{
+  private readonly IWorkspaceTypeRepository _workspaceTypeRepository;
+
+  private async Task HandleInternalPropertyValidationAsync(Operation<EditWorkspaceTypeRequest> requestedOperation, CustomContext context)
+  {
+    Context = context;
+    RequestedOperation = requestedOperation;
+
+    Thread.CurrentThread.CurrentUICulture = new CultureInfo("ru-RU");
+
+    #region paths
+
+    AddСorrectPaths(
+      new List<string>
+      {
+        nameof(EditWorkspaceTypeRequest.Name),
+        nameof(EditWorkspaceTypeRequest.Description),
+        nameof(EditWorkspaceTypeRequest.StartTime),
+        nameof(EditWorkspaceTypeRequest.EndTime),
+        nameof(EditWorkspaceTypeRequest.BookingRule),
+        nameof(EditWorkspaceTypeRequest.IsActive)
+      });
+
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.Name), new List<OperationType> { OperationType.Replace });
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.Description), new List<OperationType> { OperationType.Replace });
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.StartTime), new List<OperationType> { OperationType.Replace });
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.EndTime), new List<OperationType> { OperationType.Replace });
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.BookingRule), new List<OperationType> { OperationType.Replace });
+    AddСorrectOperations(nameof(EditWorkspaceTypeRequest.IsActive), new List<OperationType> { OperationType.Replace });
+
+    #endregion
+
+    #region Name
+
+    await AddFailureForPropertyIfAsync(
+      nameof(EditWorkspaceTypeRequest.Name),
+      x => x == OperationType.Replace,
+      new()
+      {
+        {
+          x => Task.FromResult(!string.IsNullOrEmpty(x.value?.ToString().Trim())),
```

Comment:
'x => Task.FromResult(!string.IsNullOrWhiteSpace(x.value?.ToString()))'
IsNullOrEmpty+Trim and IsNullOrWhiteSpace make the same, but with IsNullOrWhiteSpace it is shorter